databricks/koalas/tests/test_dataframe.py

```diff
@@ -2070,28 +2070,44 @@ def test_drop_duplicates(self):
         )
         kdf = ks.from_pandas(pdf)
 
-        # inplace is False
-        self.assert_eq(pdf.drop_duplicates().sort_index(), kdf.drop_duplicates().sort_index())
-        self.assert_eq(pdf.drop_duplicates("a").sort_index(), kdf.drop_duplicates("a").sort_index())
-        self.assert_eq(
-            pdf.drop_duplicates(["a", "b"]).sort_index(),
-            kdf.drop_duplicates(["a", "b"]).sort_index(),
-        )
+        for keep in ["first", "last", False]:
+            with self.subTest():
+                # inplace is False
+                self.assert_eq(
+                    pdf.drop_duplicates(keep=keep).sort_index(),
+                    kdf.drop_duplicates(keep=keep).sort_index(),
+                )
+                print("this passed")
```

Comment:
Could you remove debug prints?

Reply:
I added them this morning and forgot to take them out. :see_no_evil: